File: java/08_classes.java
class Person {
    String name;
    int age;

    Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    String greet() {
        return "hi i am " + name + " and i am " + age;
    }
}

public class ClassesExample {
    public static void main(String[] args) {
        Person me = new Person("ulas", 20);
        System.out.println(me.greet());
        System.out.println(me.name);
    }
}

File: typescript/08_classes.ts
class Person {
  name: string;
  age: number;

  constructor(name: string, age: number) {
    this.name = name;
    this.age = age;
  }

  greet(): string {
    return `hi i am ${this.name} and i am ${this.age}`;
  }
}

const me = new Person("ulas", 20);
console.log(me.greet());
console.log(me.name);
